docs/network/evm-nodes.md

```diff
@@ -17,6 +17,37 @@ You can get the kernel by importing it from a running Etherlink Smart Rollup nod
 
 Running an Etherlink EVM node on Etherlink Mainnet requires a computer with 500GB of disk space and at least 16GB RAM.
 
+## Quickstart
+
+If you don't want to see all of the options and settings for the EVM node, you can run the node quickly with these steps:
+
+1. Download the latest release of the `octez-evm-node` binary from from https://gitlab.com/tezos/tezos/-/releases.
+
+1. Run this command to start the node:
+
+   ```bash
+   octez-evm-node run observer \
+     --network <NETWORK> \
+     --history rolling:1 \
+     --data-dir <EVM_DATA_DIR> \
+     --dont-track-rollup-node \
```

Comment:
Don't think this is needed. It's part of the config

Reply:
Do you mean omitting `--dont-track-rollup-node`? When I do that and run just this command I get an error:

```
octez-evm-node run observer --network testnet --history rolling:1 --data-dir /opt/evm-quick --dont-track-rollup-node
```

I've skipped the config generation step here so `--dont-track-rollup-node` appears to be required.